3567/0/about_dice_project.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+# Implement a DiceSet Class here:
+#
+# class DiceSet
+#   code ...
+# end
+class DiceSet
+  attr_reader :values
+  def roll(num)
+    @values = []
+    (1..num).each { |_count| @values << rand(1..6) }
```

Comment:
O_o 
`@values = []` - useless row
`(1..num).each` - why we can't use Array.new?
`@values << ` - you can directly set @values -> `@values = <some_code_here>`